test/uk/gov/hmrc/pushpullnotificationsgateway/connectors/OutboundProxyConnectorSpec.scala

```diff
@@ -16,34 +16,31 @@
 
 package uk.gov.hmrc.pushpullnotificationsgateway.connectors
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future.{failed, successful}
-
 import org.mockito.captor.ArgCaptor
 import org.mockito.{ArgumentMatchersSugar, MockitoSugar}
-import org.scalatest.{Matchers, WordSpec}
-
-import play.api.LoggerLike
+import play.api.Logger
 import play.api.test.Helpers._
 import uk.gov.hmrc.http._
-import uk.gov.hmrc.play.bootstrap.http.HttpClient
-
 import uk.gov.hmrc.pushpullnotificationsgateway.config.AppConfig
 import uk.gov.hmrc.pushpullnotificationsgateway.models._
+import util.HmrcSpec
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future.{failed, successful}
 
-class OutboundProxyConnectorSpec extends WordSpec with Matchers with MockitoSugar with ArgumentMatchersSugar {
+class OutboundProxyConnectorSpec extends HmrcSpec with MockitoSugar with ArgumentMatchersSugar {
```

Comment:
MS and AMS are not required (HMRCSpec includes them). Appreciate this is not your code.

Reply:
Done